website/docs/r/vpn_gateway_nat_rule.html.markdown

```diff
@@ -58,16 +64,40 @@ The following arguments are supported:
 
 * `vpn_gateway_id` - (Required) The ID of the VPN Gateway that this VPN Gateway NAT Rule belongs to. Changing this forces a new resource to be created.
 
-* `external_address_space_mappings` - (Required) A list of CIDR Ranges which are used for external mapping of the VPN Gateway NAT Rule.
+* `external_address_space_mappings` - (Optional) A list of CIDR Ranges which are used for external mapping of the VPN Gateway NAT Rule.
+
+~> **NOTE:** `external_address_space_mappings` is deprecated and will be removed in favour of the property `external_mapping` in version 4.0 of the AzureRM Provider.
+
+* `external_mapping` - (Optional) One or more `external_mapping` blocks as documented below.
```

Comment:
Let's move `external_mapping` and `internal_mapping` up and make them required as we want those to be what people use going forward.

Reply:
Updated